com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsMaterial.cs

```diff
@@ -178,6 +179,23 @@ internal float AlbedoSaturationTolerance
         internal DebugMaterialMode DebugMaterialModeData { get; private set; }
         internal DebugVertexAttributeMode DebugVertexAttributeIndexData { get; private set; }
 
+        static class Strings
+        {
+            public const string AlbedoSettingsContainerName = "Albedo Settings";
+            public const string MetallicSettingsContainerName = "Metallic Settings";
+
+            public static readonly NameAndTooltip MaterialOverride = new() { name = "Material Override", tooltip = "Use the drop-down to select a Material property to visualize on every GameObject on screen." };
+            public static readonly NameAndTooltip VertexAttribute = new() { name = "Vertex Attribute", tooltip = "Use the drop-down to select a 3D GameObject attribute, like Texture Coordinates or Vertex Color, to visualize on screen." };
+            public static readonly NameAndTooltip MaterialValidationMode = new() { name = "Material Validation Mode", tooltip = "Debug and validate material properties." };
+            public static readonly NameAndTooltip ValidationPreset = new() { name = "Validation Preset", tooltip = "Validate through the use of a predetermined list of preset surfaces and inputs based on real world surfaces." };
```

Comment:
I think "preset" negates "predetermined" - all presets are predetermined :D. Suggestion:

"Validate using a list of preset surfaces and inputs based on real-world surfaces."